src/administrator/components/com_kunena/Controller/AttachmentsController.php

```diff
@@ -84,7 +84,7 @@ public function delete()
 
 		foreach ($cid as $id)
 		{
-			$attachment = KunenaAttachmentHelper::get($id);
+			$attachment = Helper::get($id);
```

Comment:
Just FYI, I prefer using names like `Kunena\Forum\Libraries\Attachment\AttachmentHelper` because of having just Helper makes you to make more mistakes (choose wrong class in IDE) and is pain if you need multiple classes of the same name at the same time.

Reply:
I will rename all helper to NameHelper, but first i want to have it working.